app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  alias_attribute :author, :user
+  belongs_to :user, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  after_create :update_posts_counter
+
+  def update_posts_counter
+    user.posts_counter = user.posts.count
```

Comment:
- [ ] [OPTIONAL] Instead of using `count` for the counter, you can use `increment` Ruby method like this: `author.increment!(:posts_counter)`.